test/aries-js-worker/test/verifiable/verifiable.js

```diff
@@ -30,12 +33,20 @@ const vc = `
    }
 }`
 
-// scenarios
-describe("Verifiable Store", function () {
+const restMode = 'rest'
+const wasmMode = 'wasm'
+
+describe("Verifiable Store Test", async function () {
+    await verifiableStore(newAriesREST(agentControllerApiUrl), restMode)
+    await verifiableStore(newAries())
+})
+
+async function verifiableStore(newAries, mode = wasmMode) {
     let aries
+    let modePrefix = '[' + mode + '] '
 
     before(async () => {
-        await newAries()
+        await newAries
```

Comment:
I find this pattern a bit confusing (`newAries` being a promise). Suggest minor refactor to make it clearer (eg. accept an async function as argument, or simply accept the aries object, etc.)

Reply:
we would need to pass the promise(line 40 and 41) in to some util function when we need to pass function arg. Since both create aries returns a promise, we can directly use it in func param and get the aries object. Also, this is used internally.